utils/converter_utils.py

```diff
@@ -150,3 +151,13 @@ def strip_extension(filename):
         return filename
 
 
+def attrib2dict(ob):
```

Comment:
There may be neater ways of doing this (https://stackoverflow.com/questions/61517/python-dictionary-from-an-objects-fields) - not saying you necessarily should change, just an FYI  ;-)

Reply:
Thanks! Works in a list comprehension, too.